svc-bip-api/src/main/java/gov/va/vro/bip/config/LocalEnvironmentCondition.java

```diff
@@ -0,0 +1,24 @@
+package gov.va.vro.bip.config;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+import java.util.stream.Stream;
+
+public class LocalEnvironmentCondition implements Condition {
+
+  private final String[] LOCAL_ENVS = new String[] {"local", "test"};
```

Comment:
```suggestion
  private final String[] LOCAL_ENVS = new String[] {"local", "dev", "test"};
```
I'm not confident about this suggestion. Suggested in case this is relevant: 
https://github.com/department-of-veterans-affairs/abd-vro/blob/1078dba07e8795aa8d79d354eb40449808d15983/svc-bip-api/src/main/resources/application-dev.yaml#L6

Reply:
disregard! i thought `dev` in this case mapped to something like "local dev"; however, it's actually a match for the deployment environment named `dev` (which is in the pre-prod cluster, along with deployment environments `qa` and `sandbox`).  thus, `dev` should NOT be considered a local environment.